src/views/HomePage.vue

```diff
@@ -549,6 +578,11 @@ async function changeLocale(event: MsDropdownChangeEvent): Promise<void> {
 async function validationFunction(code: Array<string>): Promise<boolean> {
   return code.length === VALID_CODE.length && code.every((value, index) => value === VALID_CODE[index]);
 }
+
+async function createStripeCard(): Promise<void> {
+  const paymentMethod: PaymentMethodResult = await stripeCardForm.value.submit();
```

Comment:
`submit()` returns a `PaymentMethodResult`, not a `PaymentMethod`. Rename the variable. It's fine if we don't check the result, it's not some critical part of the code, but it makes it a bit confusing to have `paymentMethod.paymentMethod`